README.md

```diff
@@ -1,26 +1,26 @@
-# importify — manage Haskell imports easily
+# importify — manage Haskell imports quickly
 
 [![Build Status](https://travis-ci.org/serokell/importify.svg)](https://travis-ci.org/serokell/importify)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![Coverage Status](https://coveralls.io/repos/github/serokell/importify/badge.svg?branch=master)](https://coveralls.io/github/serokell/importify?branch=master)
 [![HLint Status](https://codeclimate.com/github/serokell/importify/badges/issue_count.svg)](https://codeclimate.com/github/serokell/importify)
 
-## What this project about?
+## _Importify_ in a nutshell.
 
-`importify` tool helps you to manage import section of your Haskell project modules.
-GHC compiler can warn you about unused imports. It's a good thing to remove such
-imports. But this becomes tedious especially if you use explicit import lists.
+`importify` tool helps you to manage the import section of your Haskell project modules.
+GHC compiler can warn you about unused imports, and it's a good practice to remove such
+imports immediately. But this becomes tedious, especially if you use explicit import lists.
 
-**Importify** can remove unused imports automatically.
+_Importify_ can remove unused imports automatically.
 
-> TODO: describe what else we want it to do?
+In the future, we plan for _Importify_ to be able to add used symbols to imports automatically, so that you won't have to manage imports manually at all. To speed up _Importify_ runs, we will implement a cache server that caches each _Hackage_ package. You will be able to upload caches for your projects there as well. Also on the roadmap is functionality to convert imports between _implicit_ and _explicit_, and between _qualified_ and _unqualified_ forms. Finally, we will be adding functionality to resolve merge conflicts in import section automatically.
```

Comment:
> b. pay Serokell for private imporify cache storage on our server.

Sorry, I'm just simple programmer, I didn't look that far :D That's good point. Well, all your points make sense to me. It's just wasn't obvious to me why user might want this. But now I see why this feature is important. **2** is completely ok. Regarding **1**: we still will have option to build cache locally. If you need cache to use some refactoring tools you can just build it locally and not put this work on our server. For now it's not obvious to me why user might want to make cache of his project publicly available.